### PR TITLE
Add cut function

### PR DESCRIPTION
**Pull request by chamara-ascentic**

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1,6847 +1,8 @@
 {
   "name": "waveform-playlist",
   "version": "4.3.3",
-  "lockfileVersion": 2,
```
Comment (markuslarssonlvls) [inline package-lock.json:4]: why downgrading to version 1?

Comment (chamara-ascentic) [reply package-lock.json:4]: @markuslarssonlvls This was a mistake. Note that This commit was pushed only to let @deepfriedmind test the cut function.

Comment (markuslarssonlvls) [reply package-lock.json]: ok, I fixed it :)

File: .nvmrc [hunk 2]
```diff
@@ -0,0 +1 @@
+v16.13.0
```
Comment (deepfriedmind) [inline .nvmrc:1]: I can't run `npm i` on v16.13.0. v14.19.2 worked before this commit 🤔

Comment (markuslarssonlvls) [reply .nvmrc:1]: nvm use && npm i.  (or -> nvm use && npm ci)

Comment (deepfriedmind) [reply .nvmrc:1]: Yeah, that's what I did and I get a bunch of dependency conflicts. Does it work for you on Node v16.13.0, @chamara-ascentic?

**Pull request merged**